de/nmichael/efa/data/storage/IDataAccess.java

```diff
@@ -18,55 +18,56 @@
 
 public interface IDataAccess {
 
-    public static final int TYPE_FILE_XML = 1;
-    public static final int TYPE_EFA_REMOTE = 2;
-    public static final int TYPE_EFA_CLOUD = 3;
+    int TYPE_FILE_XML = 1;
+    int TYPE_EFA_REMOTE = 2;
+    int TYPE_EFA_CLOUD = 3;
```

Comment:
@tfyh I suppose those changes (and similar below) are by accident? I don't think we ever want to mutate these values... Please revert.